migratedb.d/01.sql

```diff
@@ -0,0 +1,20 @@
+
+DO
+$$
+DECLARE
+  sourcever INTEGER := 0;
```

Comment:
If I understand this code correctly, `sourcever` is the version the migration is going _from_. It took at least me a little while to understand this so it would be nice with a small comment to explain what this variable is doing.

Reply:
Added comment that hopefully clears this up.